source/python/idetect/model.py

```diff
@@ -279,6 +279,22 @@ class Location(Base):
         'Report', secondary=report_location, back_populates='locations')
 
 
+class TermType:
+    PERSON_TERM = 'person_term'
+    PERSON_UNIT = 'person_unit'
+    STRUCTURE_TERM = 'structure_term'
+    STRUCTURE_UNIT = 'structure_unit'
+    ARTICLE_KEYWORD = 'article_keyword'
+
+
+class ReportKeyword(Base):
+    __tablename__ = 'report_keyword'
+
+    id = Column(Integer, primary_key=True)
+    description = Column(String)
+    term_type = Column(String)
```

Comment:
(I should say that I don't have any preferences among the possible names of this thing. "Keyword" and "Term" seem like they're equivalently descriptive. It's just a little odd that there's a mismatch.)